### PR TITLE
fix: Do not use default parameter values for ie https://github.com/brix/crypto-js/pull/357

### PR DESCRIPTION
**Pull request by spencer17x**
We discussed this pr: https://github.com/brix/crypto-js/pull/357，and use this pr to fix it.

### CODE REVIEW

File: src/enc-base64url.js [hunk 1]
```diff
@@ -24,11 +24,11 @@
          *
          *     var base64String = CryptoJS.enc.Base64url.stringify(wordArray);
          */
-        stringify: function (wordArray, urlSafe=true) {
+        stringify: function (wordArray, urlSafe) {
             // Shortcuts
             var words = wordArray.words;
             var sigBytes = wordArray.sigBytes;
-            var map = urlSafe ? this._safe_map : this._map;
+            var map = urlSafe === undefined ? this._map : this._safe_map;
```
Comment (trmpowell) [inline src/enc-base64url.js]: I think you want
```
var map = urlSafe === undefined || urlSafe ? this._safe_map : this._map;
```
If `urlSafe` is `undefined`, you want this to default to `true`, so you want to use the `_safe_map` scenario. Or perhaps I'm missing something?

Comment (spencer17x) [reply src/enc-base64url.js]: If you use babel, you will find that the default value of the function parameter only needs to judge undefined.
image

Comment (spencer17x) [reply src/enc-base64url.js]: emmm...you are right, but i think maybe this way will be better?
```ts
var map = urlSafe ? this._safe_map : this._map;
````

**Comment by evanvosberg** [comment]
Already solved, but not released yet.

**Pull request closed without merging**